From 78f9441fb7c5ee91afcbb8dc10bbcbe8f48514d1 Mon Sep 17 00:00:00 2001
From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Tue, 19 Aug 2014 11:23:54 +0200
Subject: [PATCH] integration-tests: sync file systems in sync()

In sync(), also sync the file systems. This mitigates a race condition with
changing labels, which particularly surfaces with the NTFS tests.
---
 src/tests/integration-test | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/tests/integration-test b/src/tests/integration-test
index caf3231..ac4a034 100755
--- a/src/tests/integration-test
+++ b/src/tests/integration-test
@@ -203,6 +203,7 @@ class UDisksTestCase(unittest.TestCase):
             sys.stderr.write('[wait timeout!] ')
             sys.stderr.flush()
         klass.client.settle()
+        os.sync()
 
     @classmethod
     def sync_workaround(klass):
-- 
2.1.0.rc1

